src/natcap/invest/ui_server.py

```diff
@@ -152,13 +165,13 @@ def post_datastack_file():
     payload = request.get_json()
     stack_type, stack_info = datastack.get_datastack_info(
         payload['filepath'], payload.get('extractPath', None))
-    model_name = PYNAME_TO_MODEL_NAME_MAP[stack_info.model_name]
+    model_id = PYNAME_TO_MODEL_ID_MAP[stack_info.model_name]
     result_dict = {
         'type': stack_type,
         'args': stack_info.args,
         'module_name': stack_info.model_name,
-        'model_run_name': model_name,
-        'model_human_name': MODEL_METADATA[model_name].model_title,
+        'model_run_name': model_id,
+        'model_human_name': models.model_id_to_spec[model_id]['model_name'],
```

Comment:
I really like what you've done to settle on the different "names" for the different ways that an invest model can be identified ('model_id', 'model_name', 'pyname', 'alias', etc.) It would be great to carry that over to the Workbench variable names as well. I made a separate issue for that: https://github.com/natcap/invest/issues/1536

Reply:
thanks! Would appreciate any feedback on the choice of names, too - maybe 'model_name' should be 'model_title' or something more descriptive?

Reply:
I do like "model_title", but if we're never using "model_name" for anything else than I think it's just fine to use it for this purpose.